kie-internal/src/main/java/org/kie/internal/query/QueryParameterIdentifiers.java

```diff
@@ -148,4 +148,6 @@
     public static final String SUBQUERY_DEPLOYMENT = String.valueOf(idGen.getAndIncrement());
     public static final String SUBQUERY_CASE = String.valueOf(idGen.getAndIncrement());
 
+    // execution error info
+    String ERROR_DATE_LIST = String.valueOf(idGen.getAndIncrement());
```

Comment:
Add "public static final"
```suggestion
    public static final String ERROR_DATE_LIST = String.valueOf(idGen.getAndIncrement());
```